code/modules/mob/living/death.dm

```diff
@@ -49,8 +49,14 @@
 	timeofdeath = world.time
 	tod = worldtime2text()
 	var/turf/T = get_turf(src)
+	var/area/A = get_area(T)
+	if(src.mind.special_role == "Cult Master")

```

Comment:
A status effect given to the cult master that deletes itself when their stat is DEAD, and when it's deleted it does this stuff.